rb/lib/htmlproofer/enforce_img_https.rb

```diff
@@ -0,0 +1,17 @@
+class EnforceImgHTTPS < ::HTMLProofer::Check
+  def run
+    @html.css('img').each do |node|
+      img = create_element(node)
```

Comment:
`Don't include images from an insecure location!`

Maybe change to 

`Please fetch images from https:// addresses, so as to avoid mixed-content errors which hurt user confidence.`

?